client/apps/plugin-status/test/health.test.js

```diff
@@ -136,7 +163,7 @@ describe('Health View', () => {
 			</Wrapper>
 		)
 
-		const wcsSection = wrapper.find('fieldset.form-fieldset').at(2)
+		const wcsSection = wrapper.find('fieldset.form-fieldset').at(3)
```

Comment:
This is OK but it'd be better if each section had a `data-` attribute we could use to target them (just an idea; this would be too much for this PR :)).

Maybe in the meantime let's do something like this instead?

```suggestion
		const wcsSection = wrapper.find('fieldset.form-fieldset').at(wcsSectionIndex)
```